lib/ddtrace/configuration/settings.rb

```diff
@@ -244,11 +369,18 @@ def runtime_metrics(options = nil)
         end
       end
 
+      # TODO: profiler related. Ask @ivoanjo what this is for.
+      # @return [String,nil]
```

Comment:
@ivoanjo what's site, and do we want to keep it (is it deprecated)?

Reply:
`site` is used for the profiling agentless mode ([1](https://github.com/DataDog/dd-trace-rb/blob/4d77d37f7b94b845791c4695f6b75b9bef5f6256/lib/ddtrace/configuration/components.rb#L196), [2](https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/profiling/transport/http.rb#L41)). This is mostly useful for testing and validation, we don't expect customers to use this.

I would like to keep it, but am not overly attached to it. For instance, we could move it to be read directly like we do the [agentless env flag](https://github.com/DataDog/dd-trace-rb/blob/4d77d37f7b94b845791c4695f6b75b9bef5f6256/lib/ddtrace/profiling/transport/http.rb#L97) in `http.rb`, rather than having this on settings if you'd prefer to de-emphasize it.

Reply:
If you think we should keep it in `1.0` I'm more than happy to keep it.

I was just giving each of our options a hard time, and removing the ones that couldn't justify staying around yesterday, so I came across this one which I didn't know much about.

I'll add yard docs for it for now then, and keep it.